lib/template.html

```diff
@@ -223,7 +223,16 @@
 {% endif %}
 
 {% if seo_page_image %}
+  {% if page.image.height && page.image.width %}
+    "image": {
+        "@type": "ImageObject",
+        "url": {{ seo_page_image | jsonify }},
```

Comment:
My guess is that the JSON-LD checker is using this: https://developers.google.com/search/docs/data-types/articles (See `image` / `ImageObject`)